keyboards/niu_mini/keymaps/yttyx/config.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+// Disable some options to reduce firmware size
+#define NO_ACTION_TAPPING
+#define NO_ACTION_ONESHOT
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+#define NO_DEBUG
+#define NO_PRINT
+
+#endif
```

Comment:
And when you replace Lines 1 and 2, delete this line at the same time.

```suggestion
```

Reply:
Making this change created an issue whereby no layer transitions occurred. Please see general comment for more information.